shared/src/persistence/dynamo/cases/createCaseTrialSortMappingRecords.js

```diff
@@ -15,6 +18,27 @@ exports.createCaseTrialSortMappingRecords = async ({
 }) => {
   const { hybrid, nonHybrid } = caseSortTags;
 
+  const oldSortRecords = await query({
+    ExpressionAttributeNames: {
+      '#gsi1pk': 'gsi1pk',
+      '#pk': 'pk',
+    },
+    ExpressionAttributeValues: {
+      ':gsi1pk': `eligible-for-trial-case-catalog|${docketNumber}`,
+      ':pk': 'eligible-for-trial-case-catalog',
+    },
+    IndexName: 'gsi1',
+    KeyConditionExpression: '#gsi1pk = :gsi1pk AND #pk = :pk',
+    applicationContext,
+  });
+
+  if (oldSortRecords.length) {
+    await deleteCaseTrialSortMappingRecords({
+      applicationContext,
+      docketNumber,
+    });
+  }
+
```

Comment:
This is starting to look exactly like `updateCaseTrialSortMappingRecords.js` Do you think it would make enough sense to just consolidate these two into one?

Reply:
Yeah, I think you're right. Maybe only use `create...`?